luigi_pipeline/seqr_loading.py

```diff
@@ -85,11 +85,7 @@ def run(self):
 
         self.read_vcf_write_mt()
 
-    def read_vcf_write_mt(self, schema_cls=SeqrVariantsAndGenotypesSchema):
```

Comment:
rename this function to `read_input_write_mt` so it can be used both here and by the mito class